docs/recipes.rst

```diff
@@ -219,3 +219,43 @@ An example of then using this:
     print author_schema.dump(author).data
 
 This is inspired by functionality from ColanderAlchemy.
+
+Smart Nested Field
+======================================================
```

Comment:
The underline should be the same length as the title; otherwise Sphinx will raise a warning.